packages/fluent/scss/dataviz/_layout.scss

```diff
@@ -99,8 +99,8 @@
     .k-chart-tooltip td {
         width: auto;
         text-align: left;
-        padding-inline: var( --kendo-padding-x-sm, #{$kendo-padding-x-sm} );
-        padding-block: var( --kendo-padding-y-sm, #{$kendo-padding-y-sm} );
+        padding-inline: var( --kendo-padding-x-sm, #{$kendo-padding-sm-x} );
+        padding-block: var( --kendo-padding-y-sm, #{$kendo-padding-sm-y} );
```

Comment:
the css variable is `padding-x-sm`, while the scss var is `padding-sm-x`. We should probably unify the naming everywhere across components.

Reply:
will do!